cytopipe/cli/cmd.py

```diff
@@ -27,59 +27,76 @@ def run_cmd() -> None:
     -------
     None
     """
-    # checking provided cli arguments
-    cli_check(args_list=sys.argv)
+    # create args handler
+    # -- Cli Control Panel
+    args_handler = CliControlPanel(sys.argv)
```

Comment:
i stumbled for ~10 seconds wondering where you imported this class. Is there an upside to importing * from args? I think it's a bit cleaner to import everything that you use.

Reply:
My mistake! It is supposed to be the `CliControlPanel` class name!

Reply:
fixed on commit: c43041d439ed912a64a589ab61774511fe76dfb3